core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -630,28 +631,41 @@ void load()
   /**
    * @since 2.0.0
    */
-  interface ImportExecutorOptions extends ImportSourceOptions {
+  interface ImportDestinationOptions extends ImportFinalOptions {
+
+    /**
+     * Load files in the directory to the row ranges specified in the plan. The plan should contain
+     * at least one entry for every file in the directory.
+     */
+    ImportFinalOptions usingPlan(LoadPlan service);
+
+    // The javadoc below intentionally use a fully qualified class name in the value tag, otherwise
+    // it would not render properly.
     /**
      * Files are examined to determine where to load them. This examination is done in the current
-     * process using multiple threads. If this property is not set, then the client property
-     * {@code bulk.threads} is used to create a thread pool.
+     * process using multiple threads. If this method is not called, then the client property
+     * {@code bulk.threads} is used to create a thread pool. This property defaults to
+     * {@value org.apache.accumulo.core.conf.ClientDefaults#BULK_LOAD_THREADS_DEFAULT}.
      *
      * @param service
      *          Use this executor to run file examination task
      * @return ImportSourceOptions
      */
-    ImportSourceOptions usingExecutor(Executor service);
+    ImportFinalOptions examiningWith(Executor service);
 
+    // The javadoc below intentionally use a fully qualified class name in the value tag, otherwise
+    // it would not render properly.
     /**
      * Files are examined to determine where to load them. This examination is done in the current
-     * process using multiple threads. If this property is not set, then the client property
-     * {@code bulk.threads} is used to create a thread pool.
+     * process using multiple threads. If this method is not called, then the client property
+     * {@code bulk.threads} is used to create a thread pool. This property defaults to
+     * {@value org.apache.accumulo.core.conf.ClientDefaults#BULK_LOAD_THREADS_DEFAULT}.
      *
      * @param numThreads
      *          Create a thread pool with this many thread to run file examination task.
      * @return ImportSourceOptions
```

Comment:
Another less-than-useful, and now incorrect `@return` which can be removed.

Reply:
deleted